docker/Dockerfile

```diff
@@ -1,27 +1,32 @@
 FROM jupyter/base-notebook:python-3.8.6
-LABEL maintainer="pyvista Developers"
+LABEL maintainer="PyVista Developers"
 
 USER root
 RUN apt-get update \
  && apt-get install  -yq --no-install-recommends \
     libgl1-mesa-glx \
     libglu1-mesa \
+    libfontconfig1 \
+    libxrender1 \
     xvfb \
  && apt-get clean && rm -rf /var/lib/apt/lists/*
 USER jovyan
 
-# setup the itkwidgets conda environment
-COPY environment.yml labextensions.txt /tmp/
-RUN conda env update --name base --file /tmp/environment.yml
-RUN conda run conda install -y nodejs
-RUN conda run jupyter labextension install $(cat /tmp/labextensions.txt)
+COPY pyvista/ /build-context/pyvista/
+COPY setup.py /build-context/
+COPY README.rst /build-context/
+COPY LICENSE /build-context/
+COPY requirements*.txt /build-context/
+WORKDIR /build-context/
+RUN pip install -r requirements_docs.txt -r requirements_test.txt
+RUN pip install .
 
-RUN pip install ipyvtklink==0.2.1
+WORKDIR $HOME
 
 # allow jupyterlab for ipyvtk
 ENV DISPLAY=:99.0
 ENV JUPYTER_ENABLE_LAB=yes
 ENV PYVISTA_USE_IPYVTK=true
 
-# modify the CMD and start a background server first
+# modify the CMD to start a background Xvfb process first
```

Comment:
Note:
If we use the OSMesa VTK library published at https://github.com/pyvista/pyvista-wheels, we won't need xvfb.

Reply:
Ran some quick benchmarks on a headless machine:
#### Vanilla VTK wheel Python 3.8 9.1.0
```
$ time xvfb-run python -c "import pyvista;[pyvista.Sphere().plot(off_screen=True) for _ in range(100)]"

real	0m8.072s
user	0m14.052s
sys	0m0.895s
```

#### Custom VTK wheel Python 3.8 9.1.0 built using OSMesa
```
time python -c "import pyvista;[pyvista.Sphere().plot(off_screen=True) for _ in range(100)]"

real	0m5.410s
user	0m13.009s
sys	0m0.636s
```

That's a significant improvement (~50% speedup).

Reply:
This is actually really cool - those wheels are going to be incredibly useful for me moving forward

IMO: If using these wheels, that's another reason not to use conda